draft-ietf-ivy-network-inventory-yang.md

```diff
@@ -302,31 +316,19 @@ Different types of components can be distinguished by the class of component. Th
 
 Other types of components can be defined in other documents, together with the associated YANG identity and the rationale for managing them as components from an inventory perspective.
 
-Attributes related to specific class of component can be found in the component-specific-info structure.
-
 The identity definition of additional types of "ne-type" and "non-
 hardware" identity of component are outside the scope of this
 document and could be defined in application- and technology-specific companion augmentation data models, such as
 {{?I-D.ietf-ivy-network-inventory-software}}.
 
+[comment]: # The ne-specific-info and component-specific-info containers are defined as augmentation targets for the attributes related to specific network element types or component classes to be defined in this or in other application- and technology-specific companion augmentation data models.
+
```

Comment:
At the moment in the model we have component specific-info only:
                 ```
+--ro chassis-specific-info
                 +--ro slot-specific-info
                 +--ro board-specific-info
                 +--ro port-specific-info
```
I would avoid to write ne-specific-info (that are not present in the model) and component-specific-info, but just use "specific info " only. So the text could sound " The specific-info containers are defined as augmentation targets ...."